Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -1,4 +1,6 @@
 using Content.Client.UserInterface.Systems.Chat.Controls;
+using Content.Shared.CCVar;
+using Content.Shared._EE.CCVars;
```

Comment:
Using directive is unnecessary.
```suggestion
using Content.Client.UserInterface.Systems.Chat.Controls;
using Content.Shared._EE.CCVars;
```